contents/teams/marketing/objectives.mdx

```diff
@@ -1,92 +1,66 @@
-## Product focus: Web analytics
-
-- **Rationale:** We want to focus most of our marketing efforts around a single product, giving the team greater focus. We're choosing web analytics because it's recently out of beta and has strong adoption already.
-
-- **Owned by:** Lior and Ian
-
-- **Contributors:** Andy and others where needed
+## Newsletter
 
+- **Rationale:** Our newsletter is the centerpiece of our content. We have published really good content and want to continue to do this.
+- **Owned by:** Ian
+- **Contributors:** EVERYONE
 - **What we're shipping:**
-  - Great web analytics docs
-  - Comparisons for main competitors
-  - Alternatives guides for main competitors
-  - Better privacy docs and advice (GDPR consent etc.)
-  - Proper web vs product analytics guide
-  - Loads of relevant tutorials and SEO content
-  - Social ads for web analytics
-
+ - Publish at least one newsletter every two weeks
+ - Test a new format that can potentially help us publish more regularly
+ - Every issue gets a James Twitter post and a LinkedIn post
+ - Bijan and Danilo publish a newsletter
 - **We'll know we're successful when:**
-  - We see increasing usage of web analytics
-  - More people choosing web analytics at onboarding
-  - Growing word of mouth for PostHog as a web analytics tool (brand mentions)
-
-## Content focus: Newsletter
+  - Our subscriber growth rises to over 8k per month.
+  - Everyone feels confident in shipping a newsletter issue. 
+  - We see a significant growth in Twitter and LinkedIn as sources of traffic.
 
-- **Rationale:** We've built a successful newsletter, but we need to aim higher. We want to ship more ambitious and original articles. Our ultimate long-term goal is people should talk about Product for Engineers in the same, or similar, terms as Lenny's Newsletter. A "default" newsletter that is synonmous with quality.
-
-- **Owned by:** Andy
-
-- **Contributors:** Ian, Lior, James Hawkins and anyone else who wants to get involved
+## Podcast
 
+- **Rationale:** TBC
```

Comment:
@bijanbwb for you to flesh out

Reply:
Can I push to this branch or should I create a separate PR?

Reply:
Push to this branch :)